apps/product/tasks.py

```
@@ -1,207 +1,982 @@
+                    date_from = date_from1.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+                else:
+                    date_from = (date_from1 + timedelta(days=3)).strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+
+@app.task
+def update_ozon_orders():
+    
+    FBO_URL = "https://api-seller.ozon.ru/v2/posting/fbo/list"
+    FBS_URL = "https://api-seller.ozon.ru/v2/posting/fbs/list"
+    try:
+        date_from = ProductOrder.objects.filter(marketplace_type="ozon").latest('date').date.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+    except:
```

Ruff

Do not use bare `except`

(E722)